Hunting Queries/Microsoft 365 Defender/Campaigns/Abuse.ch Recent Threat Feed.yaml

```diff
@@ -25,6 +25,7 @@ query: |
       [@"https://bazaar.abuse.ch/export/csv/recent/"]
       with (format = "txt"))
       | where report !startswith '#'
+      | extend report = replace_regex(report,@"""",@"")
```

Comment:
Use trim function to perform this operation or normal replace would work as well.